### ISSUE TITLE
should have an option of Accessibility mode for the blind person

### ISSUE THREAD

**Issue by beingsmart3** [ahmedkaludi/accelerated-mobile-pages#1647]
should have an option of Accessibility mode for the blind persons

Ref:  https://secure.helpscout.net/conversation/510637824/20676/

**Comment by MARQAS** [comment]
Ahmed here,
We've kind of taken care of this in #1691
